info.yml

```diff
@@ -122,6 +122,24 @@ configuration:
         values:
             type: dict
 
+    published_file_matching_fields:
+        type: list
+        description: "The Published File fields to use when looking for the publish history. They are
+                      used to match the current Published File data against other Published Files. By default,
+                      the app will look for Published Files with the same project, task, entity, name and
+                      published file type."
+        values:
+            type: str
+        allows_empty: False
+        default_value:
+            ["project", "task", "entity", "name", "published_file_type"]
+    published_file_fields:
```

Comment:
Since there is a `publish_filters` setting already, it would be better to follow the convention and have `publish_fields`.

Reply:
I'll keep it as this since it already existed in `tk-multi-breakdown2` with that name.